chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/job/urlStatus/QuestionableUrlReportGenerator.java

```diff
@@ -131,11 +136,12 @@ public int compare(final FailedUrlResult o1, final FailedUrlResult o2) {
     }
 
     private List<FailedUrlResult> filterUrls(List<FailedUrlResult> badUrls, JobExecutionContext jobContext) {
-        String acbSpecific = jobContext.getMergedJobDataMap().getString("acbSpecific");
-        if (!StringUtils.isEmpty(acbSpecific) && BooleanUtils.toBoolean(acbSpecific)) {
+        if (isAcbSpecific(jobContext)) {
             List<Long> acbIds = getSelectedAcbIds(jobContext);
             return badUrls.stream()
                 .filter(badUrl -> isUrlRelatedToAcbs(badUrl, acbIds))
+                .filter(badUrl -> isNotListingUrl(badUrl) || isUrlRelatedTo2015Edition(badUrl))
+                .filter(badUrl -> isNotListingUrl(badUrl) || isUrlRelatedToActiveListing(badUrl))
```

Comment:
Should these be combined into one?
```suggestion
                .filter(badUrl -> isNotListingUrl(badUrl) || isUrlRelatedTo2015Edition(badUrl) || isUrlRelatedToActiveListing(badUrl))
```

The line above is filtering based on ACBs based on the job definition, and the one below is about the business logic around the status codes. These two are both about what types of URLs should be in the report, which feels like a different check to the ACB & code checks, but they both feel like the same kind of check as each other, which is why I wonder if they can be combined into one line